StripePayments/StripePayments/Source/API Bindings/Models/Shared/LinkSettings.swift

```diff
@@ -61,6 +64,8 @@ import Foundation
         let webviewOption = PopupWebviewOption(rawValue: response["link_popup_webview_option"] as? String ?? "")
         let passthroughModeEnabled = response["link_passthrough_mode_enabled"] as? Bool ?? false
         let disableSignup = response["link_mobile_disable_signup"] as? Bool ?? false
+        // Should default to true in case we remove the server-side flag someday
+        let useRebrand = response["link_2024_rebrand_m1"] as? Bool
```

Comment:
The comment indicates defaulting to true, but there doesn't seem to be a default value?

Reply:
Fixed, thanks!